Good-Bids/goodbids
# [#216] WooCommerce Emails

# Summary

This styles the default WooCcmmerce email template when a new site is created. This also refactors the terms and conditions and privacy link so they are global and can be pull into the footer and the email template. I also simplified the new site default logo to pull from the main site logo vs being hardcoded. 

## Issues

* #216

## Testing Instructions

1. Create a new site.
2. Go to WooCommerce -> Settings -> Emails
3. Scroll to the bottom and click Click here to "preview your email template"
   - Header color is dark green.
   - Text is dark green.
   - Email footer logo and links.
5. Make sure it matches the screenshot.

## Screenshots

<img width="672" alt="Screenshot 2024-01-18 at 2 00 11 PM" src="https://github.com/Good-Bids/goodbids/assets/91974372/78a7a9c6-cf67-435b-a451-86949afc8627">
